html/includes/functions.inc.php

```diff
@@ -1349,6 +1349,15 @@ function get_oxidized_nodes_list()
         }
         echo "
         </td>
+        ";
+        if ($config['force_ip_to_sysname'] === false) {
```

Comment:
Please use `Config::get('force_ip_to_sysname')` instead of accessing the global variable.

Reply:
Done